pkgs/development/tools/packet/default.nix

```diff
@@ -1,25 +1,25 @@
-# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
-{ lib, buildGoPackage, fetchgit }:
+{ lib, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage rec {
   pname = "packet";
   version = "2.2.2";
 
   goPackagePath = "github.com/ebsarr/packet";
 
-  src = fetchgit {
+  src = fetchFromGitHub {
+    owner = "ebsarr";
+    repo = "packet";
     rev = "v${version}";
-    url = "https://github.com/ebsarr/packet";
     sha256 = "18n8f2rlab4icb28k1b9gnh30zy382v792x07fmcdqq4nkw6wvwf";
```

Comment:
Did you check if we need to update this?

Reply:
no need to update it, but I recalculated it to the sri-form.